src/pages/RequestCallPage.js

```diff
@@ -38,8 +38,8 @@ import networkPropTypes from '../components/networkPropTypes';
 const propTypes = {
     ...withLocalizePropTypes,
 
-    /** The personal details of the person who is logged in */
-    myPersonalDetails: personalDetailsPropType.isRequired,
+    /** Personal details of all the users */
```

Comment:
In `AdditionalDetailsStep.js` you have this at the end of the same comment: `, including current user`

Probably should be consistent - I think it makes sense to keep `, including current user` in these comments